jax/tree_util.py

```diff
@@ -39,6 +39,7 @@
 # See PEP 484 & https://github.com/google/jax/issues/7570
 
 from jax._src.tree_util import (
+  MAP_TO_NONE as MAP_TO_NONE,
```

Comment:
Maybe export class `_MapToNone` as well since it may be used in writting type signatures.